sass/components/_tooltip.scss

```diff
@@ -10,7 +10,6 @@
         font-size: 13px;
         font-weight: 500;
         padding: 5px 10px 6px;
-        word-break: break-word;
```

Comment:
Before removing this rule, the unflag tool tip was breaking inappropriately 👇 
![image](https://user-images.githubusercontent.com/25670682/81955869-ebe95c00-960a-11ea-84e0-6d70087ba24d.png)

The unflag works fine after removing this rule, and it doesn't seem to cause regressions in other tooltips (samples 👇)
![image](https://user-images.githubusercontent.com/25670682/81956042-23f09f00-960b-11ea-887d-b16e002a41c5.png)
![image](https://user-images.githubusercontent.com/25670682/81956094-336fe800-960b-11ea-8a1e-bdc41bc5c394.png)
![image](https://user-images.githubusercontent.com/25670682/81956139-3ec31380-960b-11ea-8745-10bbf46f73cc.png)